src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveLoadVectorFirstFaultingTest.template

```diff
@@ -262,22 +261,22 @@ namespace JIT.HardwareIntrinsics.Arm
         public void ConditionalSelect_ZeroOp()
         {
             TestLibrary.TestFramework.BeginScenario(nameof(ConditionalSelect_ZeroOp));
-            ConditionalSelectScenario(_mask, ref _fld1, {Op1VectorType}<{RetBaseType}>.Zero);
-            
+            ConditionalSelectScenario(_mask, _dataTable.inArray1Ptr, {Op1VectorType}<{RetBaseType}>.Zero);
```

Comment:
why this change?

Reply:
> why this change?

It's tied to other changes in the file, let me try to explain:

1.  In case of this template `{Op1BaseType}` and `{Op2BaseType}` are the types of the `mask` and `address` arguments accordingly. Prior to this change `ValidateConditionalSelectResult` used to accept a vector/array of `{Op1BaseType}` elements as the first `firstOp` (data) argument. This was correct as long as `{Op1BaseType}` and `{Op2BaseType}` are same which used to be true any more. This change adds `LoadVector*FirstFaulting` APIs for which this doesn't hold true. This explains changes at lines `324`-`339` of this file.
2. Due to the change `1` in the signature of `ValidateConditionalSelectResult`, the second parameter must be passed as `{Op2BaseType*}` which leads to the change at line `279`.
3. Since `{Op1BaseType}` and `{Op2BaseType}` may differ now, `_fld1` which has `{Op1VectorType}<{RetBaseType}>` type can't be passes to `ValidateConditionalSelectResult`. To overcome this, this change replace it with `_dataTable.inArray1Ptr` of `void *` type at lines `264`-`270`. (Also `_fld1` is assigned a return value of `LoadVector*FirstFaulting` at line `234` which is perfectly fine).
4. Because of `3`, the signature of `ConditionalSelectScenario` was changed at line `274`.

Please let me know if you would like to get more details on this or think this should be implemented another way.

Reply:
Now I see this, I think `_fld1` was passed uninitialized in https://github.com/dotnet/runtime/pull/105595. cc: @TIHan . `inArray1Ptr` is the parameter that should be passed here as you pointed out.